Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -544,14 +544,22 @@ void MainWidget::initialize_menubar(GUI::Window& window)
         "Flip Image &Vertically", g_icon_bag.edit_flip_vertical, [&](auto&) {
             auto* editor = current_image_editor();
             VERIFY(editor);
-            editor->image().flip(Gfx::Orientation::Vertical);
+            auto image_flip_or_error = editor->image().flip(Gfx::Orientation::Vertical);
+            if (image_flip_or_error.is_error()) {
+                GUI::MessageBox::show_error(&window, image_flip_or_error.error().string_literal());
```

Comment:
For all these error message boxes, it would be nice if you displayed a formatting string like "Failed to flip image: <error>" instead of just "<error>", since the error is likely to just be "Out of memory" which isn't very explanatory by itself.